tests/mr/device/arena_mr_tests.cpp

```diff
@@ -574,10 +574,10 @@ TEST_F(ArenaTest, DumpLogOnFailure)  // NOLINT
     std::size_t num_threads{4};
     threads.reserve(num_threads);
     for (std::size_t i = 0; i < num_threads; ++i) {
-      threads.emplace_back(std::thread([&] {
```

Comment:
Unrelated improvement?

Reply:
yes fixing a clang-tidy complaint.

Reply:
Are you saying you want this moved out of this PR?